client/landing/stepper/declarative-flow/plugin-bundle-data.ts

```diff
@@ -95,6 +95,13 @@ export const bundleStepsSettings: BundleStepsSettings = {
 		},
 		checkForActivePlugins: [ 'woocommerce' ],
 	},
+	sensei: {
+		customSteps: [],
+		checkForActivePlugins: [ 'sensei-lms' ],
+		endFlow: ( { adminUrl, exitFlow } ) => {
+			return exitFlow( `${ adminUrl }admin.php?page=sensei_setup_wizard` );
```

Comment:
Completely a different problem for a different pr and repo, but the setup wizard encourages you to use the Course theme even if you already have the Course theme activated.

Reply:
Good catch, @dsas!
Sorry that I didn't mention it previously in this PR, but it's already in our backlog to be changed in the next few days! https://github.com/Automattic/sensei/issues/6350